code/lib/core-server/src/utils/server-address.ts

```diff
@@ -46,3 +46,10 @@ export const getServerPort = (port?: number, { exactPort }: PortOptions = {}) =>
 export const getServerChannelUrl = (port: number, { https }: { https?: boolean }) => {
   return `${https ? 'wss' : 'ws'}://localhost:${port}/storybook-server-channel`;
 };
+
+const getLocalIp = () => {
+  const allIps = Object.values(os.networkInterfaces()).flat();
+  const allFilteredIps = allIps.filter((ip) => ip && ip.family === 'IPv4' && !ip.internal);
+
+  return allFilteredIps.length ? allFilteredIps[0]?.address : '0.0.0.0';
```

Comment:
I am curious whether returning `0.0.0.0` makes sense. Can the Storybook server be reached in the browser when `0.0.0.0` is used? If not, I would rather return `null/undefined` and handle the case `getServerAddresses` function. It should return `networkAddress: undefined`. In this case, the network address shouldn't be logged at all to the user's console, after the server starts.

Reply:
I'm not sure which returned value is better. The Storybook server is reachable via `0.0.0.0` in my browsers. Maybe it also depends on how to interpret 'On your network' in the log.